packages/varlet-ui/src/date-picker/DatePicker.vue

```diff
@@ -513,8 +513,9 @@ export default defineComponent({
       rangeDate.value = formatDateList.filter((date) => date !== 'Invalid Date')
     }
 
-    function dateInit(value: string) {
-      const formatDate = dayjs(value).format('YYYY-MM-D')
+    function dateInit(value: string | undefined) {
+      const handleValue = value ? dayjs(value) : dayjs()
+      const formatDate = dayjs(handleValue).format('YYYY-MM-D')
```

Comment:
If the value is undefined or an empty string, `handleValue` will be `dayjs()`. This code `dayjs(dayjs())`  seems not necessary.

It looks like no differences.
![image](https://github.com/user-attachments/assets/a459cd6c-f4cd-4cd2-af54-18f65c9e36bd)